packages/api/src/apollo.ts

```diff
@@ -32,6 +32,7 @@ import { ClaimsToSet, RequestContext, ResolverContext } from './resolvers/types'
 import ScalarResolvers from './scalars'
 import typeDefs from './schema'
 import { batchGetHighlightsFromLibraryItemIds } from './services/highlights'
+import { batchGetJustReadFeedItems } from './services/just_read_feed'
```

Comment:
I think `home_feed` will make the most sense for this. If we want we can even just call it `Home` and `Home` has `HomeItem`s